### ISSUE TITLE
Layer Create: sometimes doesn't have a GeoGig repo generated

### ISSUE THREAD

**Issue by emilyashley** [MapStory/mapstory#935]
Was reviewing PR #928 and noticed this pre-existing bug. 

![screen shot 2017-08-09 at 11 32 44 am](https://user-images.githubusercontent.com/15912063/29140442-50e9022c-7cff-11e7-8ab1-061492f7bfff.png)

Sometimes upon Layer Creation, it 400's and the server logs: `No GeoGig repository found with NAME or ID: anotherlayer`

I tried this 21 times, and 7 of them were successful. 14 were unsuccessful. 
It works 1/3rd of the time.

Server logs below for anyone interested. It might have something to do with how I'm trying to create them back to back.... maybe it needs more time between layer creation. 20 tries, only instances 1, 2, 4, 5, 8, and 10 worked. The more I tried, the less successful they seem

I waited 4 minutes and try 21 worked
So yeah, I'm either thinking random or time

**Comment by emilyashley** [comment]
```
Emilys-MBP:mapstory emilyashley$ docker attach mapstory_django_1
geometry
time
[09/Aug/2017 14:20:13] "GET /layers/acls HTTP/1.1" 200 98
[09/Aug/2017 14:20:13] "GET /layers/acls HTTP/1.1" 200 303
[09/Aug/2017 14:20:18] "GET /layers/acls HTTP/1.1" 200 303
[09/Aug/2017 14:20:21] "GET /layers/acls HTTP/1.1" 200 303
[09/Aug/2017 14:20:21] "POST /layers/create HTTP/1.0" 201 82
geometry
time
[09/Aug/2017 14:20:37] "GET /layers/acls HTTP/1.1" 200 315
[09/Aug/2017 14:20:39] "GET /layers/acls HTTP/1.1" 200 315
[09/Aug/2017 14:20:41] "POST /layers/create HTTP/1.0" 201 84
geometry
time
[09/Aug/2017 14:20:57] "GET /layers/acls HTTP/1.1" 200 328
:No GeoGig repository found with NAME or ID: onmaster3
[09/Aug/2017 14:20:58] "POST /layers/create HTTP/1.0" 400 119
geometry
time
[09/Aug/2017 14:21:26] "GET /layers/acls HTTP/1.1" 200 328
[09/Aug/2017 14:21:28] "GET /layers/acls HTTP/1.1" 200 328
[09/Aug/2017 14:21:29] "POST /layers/create HTTP/1.0" 201 84
geometry
time
[09/Aug/2017 14:21:56] "GET /layers/acls HTTP/1.1" 200 341
[09/Aug/2017 14:21:58] "GET /layers/acls HTTP/1.1" 200 341
[09/Aug/2017 14:22:00] "POST /layers/create HTTP/1.0" 201 84
geometry
time
[09/Aug/2017 14:22:22] "GET /layers/acls HTTP/1.1" 200 354
:No GeoGig repository found with NAME or ID: onmaster6
[09/Aug/2017 14:22:23] "POST /layers/create HTTP/1.0" 400 119
geometry
time
[09/Aug/2017 14:22:50] "GET /layers/acls HTTP/1.1" 200 354
:No GeoGig repository found with NAME or ID: onmaster7
[09/Aug/2017 14:22:51] "POST /layers/create HTTP/1.0" 400 119
geometry
time
[09/Aug/2017 14:23:15] "GET /layers/acls HTTP/1.1" 200 354
[09/Aug/2017 14:23:17] "GET /layers/acls HTTP/1.1" 200 354
[09/Aug/2017 14:23:19] "POST /layers/create HTTP/1.0" 201 84
geometry
time
[09/Aug/2017 14:23:37] "GET /layers/acls HTTP/1.1" 200 367
:No GeoGig repository found with NAME or ID: onmaster9
[09/Aug/2017 14:23:38] "POST /layers/create HTTP/1.0" 400 119
geometry
time
[09/Aug/2017 14:24:11] "GET /layers/acls HTTP/1.1" 200 367
[09/Aug/2017 14:24:13] "GET /layers/acls HTTP/1.1" 200 367
[09/Aug/2017 14:24:15] "POST /layers/create HTTP/1.0" 201 86
geometry
time
[09/Aug/2017 14:24:32] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster11
[09/Aug/2017 14:24:33] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:24:53] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster12
[09/Aug/2017 14:24:54] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:25:11] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster13
[09/Aug/2017 14:25:12] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:25:32] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster14
[09/Aug/2017 14:25:33] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:26:13] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster15
[09/Aug/2017 14:26:14] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:26:33] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster16
[09/Aug/2017 14:26:34] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:27:11] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster17
[09/Aug/2017 14:27:12] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:28:12] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster18
[09/Aug/2017 14:28:13] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:28:34] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster19
[09/Aug/2017 14:28:35] "POST /layers/create HTTP/1.0" 400 121
geometry
time
[09/Aug/2017 14:28:52] "GET /layers/acls HTTP/1.1" 200 381
:No GeoGig repository found with NAME or ID: onmaster20
[09/Aug/2017 14:28:53] "POST /layers/create HTTP/1.0" 400 121

—waited 4 minutes —

geometry
time
[09/Aug/2017 14:32:11] "GET /layers/acls HTTP/1.1" 200 98
[09/Aug/2017 14:32:11] "GET /layers/acls HTTP/1.1" 200 381
[09/Aug/2017 14:32:13] "GET /layers/acls HTTP/1.1" 200 381
[09/Aug/2017 14:32:15] "POST /layers/create HTTP/1.0" 201 86
```

**Comment by emilyashley** [comment]
confirmed by tyler during the tech sync today 
![pasted image at 2017_08_24 11_14 am](https://user-images.githubusercontent.com/15912063/29683542-569bf7a2-88d4-11e7-9832-acdd58af0065.png)

**Comment by jonpmarino** [comment]
related but separate. @lhcramer and I are noticing that layer creating for polygon and line layers seem to fail with the same error message in the screenshot above, but point layers create.

**Comment by emilyashley** [comment]
Addressed w/ #1013
